src/Blocks/Covers.php

```diff
@@ -180,13 +178,11 @@ public function prepare_data(array $fields): array
      */
     public static function get_covers(array $fields): array
     {
-        $cover_type = $fields['cover_type'] ?? self::CONTENT_COVER_TYPE;
+        $cover_type = $fields['cover_type'];
         $covers = [];
 
         if (self::TAKE_ACTION_COVER_TYPE === $cover_type) {
             $covers = self::populate_posts_for_act_pages($fields);
-        } elseif (self::CONTENT_COVER_TYPE === $cover_type) {
-            $covers = self::populate_posts_for_cfc($fields);
```

Comment:
I think that this function can be removed, it's not used anywhere else

Reply:
I'm talking about `populate_posts_for_cfc`

Reply:
Ohh, I see! You're 100% right, thanks!